Assets/Styles/abstracts/_helpers.scss

```diff
@@ -0,0 +1,55 @@
+// -----------------------------------------------------------------------------
+// This file contains CSS helper classes.
+// -----------------------------------------------------------------------------
+
+@import "mixins";
+
+/**
+ * Clear inner floats
+ */
+.clearfix::after {
+    clear: both;
+    content: "";
+    display: table;
+}
+
+@if variable-exists(debug-mode) and $debug-mode {
+    * {
+        // Required to override existing styles for debugging.
+        // stylelint-disable declaration-no-important
+        background: #000 !important;
+        color: #0f0 !important;
+        outline: solid #f00 1px !important;
+        // stylelint-enable declaration-no-important
+    }
+}
+
+/**
+ * Hide element while making it readable for screen readers
+ * Shamelessly borrowed from HTML5Boilerplate:
```

Comment:
lel

Reply:
Who wrote this? 
![image](https://user-images.githubusercontent.com/5919557/148584268-c066c54c-d14b-43b5-8c29-b6d6907223fe.png)